src/luacheck/love_standard.lua

```diff
@@ -48,14 +50,18 @@ local love = {
          "setDistanceModel","setDopplerScale","setOrientation","setPosition","setVelocity",
          "setVolume","stop"),
 
+      data = standards.def_fields("compress","decode","decompress","encode","getPackedSize","hash","newByteData",
+         "newDataView","pack","unpack"),
+
       event = standards.def_fields("clear","poll","pump","push","quit","wait"),
 
       filesystem = standards.def_fields("append","areSymlinksEnabled","createDirectory","exists",
          "getAppdataDirectory","getDirectoryItems","getIdentity","getLastModified",
          "getRealDirectory","getRequirePath","getSaveDirectory","getSize","getSource",
          "getSourceBaseDirectory","getUserDirectory","getWorkingDirectory","init","isDirectory",
          "isFile","isFused","isSymlink","lines","load","mount","newFile","newFileData","read",
-         "remove","setIdentity","setRequirePath","setSource","setSymlinksEnabled","unmount","write"),
+         "remove","setIdentity","setRequirePath","setSource","setSymlinksEnabled","unmount","write",
+         "getInfo","setCRequirePath","getCRequirePath"),
 
```

Comment:
I noticed reviewing this that the `love.font` module is missing which should have the following fields:
```lua
font = standards.def_fields("newImageRasterizer", "newGlyphData", "newTrueTypeRasterizer", "newBMFontRasterizer", "newRasterizer"),
```